src/schemas/types.json

```diff
@@ -160,6 +160,50 @@
             "default": 0,
             "minimum": 0,
             "maximum": 9999
+        },
+        "Dimensions": {
+            "type": "object",
+            "properties": {
+                "width": {
+                    "type": "integer",
+                    "minimum": 1
+                },
+                "height": {
+                    "type": "integer",
+                    "minimum": 1
+                }
+            },
+            "required": [ "width", "height" ]
+        },
+        "Image": {
+            "type": "object",
+            "properties": {
+                "path": {
```

Comment:
ideally we could reuse this for both web and packaged images. Can we call this `uri` which is appropriate for both relative paths, e.g. `./foo/image.png` and also hosted images, e.g. `https://foo.com/bar.png`